lib/web.mlx

```diff
@@ -188,6 +188,75 @@ let getting_started ~install_url () =
 "$ cd hello_world"
 "$ dune pkg lock"
 "$ dune exec hello_world"
+    </Code>
+    <h3 class_="mt-10 mb-2.5">"Editor Configuration"</h3>
+    <p class_="text-white">
+    "Editor integration for OCaml is done using the tool "
```

Comment:
As it is a getting started, I wouldn't go deep into the detail. Maybe adding the explanation to the FAQ or in another part of the website.